packages/design-picker/src/components/style.scss

```
@@ -173,3 +165,36 @@
 		}
 	}
 }
+
+// dark theme styles
+.design-picker.design-picker--theme-dark {
+	.design-picker__option-name {
+		color: var( --studio-white );
+	}
+	.design-picker__design-option {
+		.design-picker__image-frame {
+			border-color: var( --studio-gray-40 );
+		}
+		&:hover,
+		&:focus {
+			.design-picker__image-frame {
+				border-color: var( --studio-white );
+			}
+		}
+	}
+}
+
+// light theme styles
+.design-picker.design-picker--theme-light {
+	.design-picker__option-name {
+		color: var( --studio-gray-40 );
+	}
+	.design-picker__design-option {
```

Semi-nit, but:

similarly to how I [suggested moving the CSS rules the text color](https://github.com/Automattic/wp-calypso/pull/52162#discussion_r620098964) for both dark and light themes, I think that it would be better to have the `border-color` CSS rule for the light theme here too:

```suggestion
	.design-picker__design-option {
		.design-picker__image-frame {
			border-color: var( --studio-gray-5 );
		}
```

With this change applied, the CSS rule (currently at line 102) that says `border: 1px solid var( --studio-gray-5 );` could be replaced by `border: 1px solid;`